components/vars/css/components/spectrum-button.css

```diff
@@ -1193,22 +1200,22 @@
   --spectrum-button-primary-xl-text-font-family: var(--spectrum-alias-body-text-font-family);
   --spectrum-button-primary-xl-text-font-style: var(--spectrum-global-font-style-regular);
   --spectrum-button-primary-xl-text-font-weight: var(--spectrum-global-font-weight-bold);
-  --spectrum-button-primary-xl-text-line-height: var(--spectrum-alias-button-text-line-height);
+  --spectrum-button-primary-xl-text-line-height: var(--spectrum-alias-component-text-line-height);
   --spectrum-button-primary-xl-text-letter-spacing: var(--spectrum-global-font-letter-spacing-none);
   --spectrum-button-primary-xl-text-transform: none;
   --spectrum-button-primary-xl-border-size: var(--spectrum-alias-border-size-thick);
   --spectrum-button-primary-xl-icon-size: var(--spectrum-alias-workflow-icon-size-xl);
-  --spectrum-button-primary-xl-text-size: var(--spectrum-alias-item-text-size-xl);
   --spectrum-button-primary-xl-icon-padding-top: var(--spectrum-alias-item-icon-padding-top-xl);
   --spectrum-button-primary-xl-icon-padding-bottom: var(--spectrum-alias-item-icon-padding-bottom-xl);
-  --spectrum-button-primary-xl-icon-gap: var(--spectrum-global-dimension-size-125);
+  --spectrum-button-primary-xl-icon-gap: var(--spectrum-alias-item-workflow-icon-gap-l);
+  --spectrum-button-primary-xl-text-size: var(--spectrum-alias-item-text-size-xl);
   --spectrum-button-primary-xl-text-padding-top: var(--spectrum-alias-item-text-padding-top-xl);
   --spectrum-button-primary-xl-text-padding-bottom: var(--spectrum-alias-item-text-padding-bottom-xl);
   --spectrum-button-primary-xl-height: var(--spectrum-alias-item-height-xl);
   --spectrum-button-primary-xl-padding-left: var(--spectrum-alias-item-rounded-workflow-padding-left-xl);
   --spectrum-button-primary-xl-padding-right: var(--spectrum-alias-item-rounded-padding-xl);
   --spectrum-button-primary-xl-border-radius: var(--spectrum-alias-item-rounded-border-radius-xl);
-  --spectrum-button-primary-xl-min-width: var(--spectrum-global-dimension-size-1350);
+  --spectrum-button-primary-xl-min-width: var(--spectrum-global-dimension-size-1250);
```

Comment:
maybe double check that this is `        "spectrum-button-primary-xl-min-width": "100px",`

Reply:
This file is generated from DNA, not manually maintained.

And yes, `--spectrum-global-dimension-size-1250: 100px;`, you can check this by looking at `vars/css/scales/spectrum-medium.css`